<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>675</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Running Gait Analysis</string>
  </property>
  <widget class="QPushButton" name="processButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QLabel" name="sideLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SIDE VIEW - FRAME: </string>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>370</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BACK VIEW - FRAME:</string>
   </property>
  </widget>
  <widget class="QSlider" name="backViewSlider">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>400</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sideViewSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="metricSelectComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Torso Lean</string>
   </property>
   <property name="placeholderText">
    <string>Empty</string>
   </property>
   <item>
    <property name="text">
     <string>Torso Lean</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Elbow Angle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Center of Mass Displacement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Knee Flexion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tibia Angle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feet Strike</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hip Extension</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pelvic Drop</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="metricDescriptionText">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>481</width>
     <height>192</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;/mnt/c/Users/rados/Documents/BP/openpose/openpose/outputs/traintrack/123_side2_new/  /mnt/c/Users/rados/Documents/BP/openpose/openpose/outputs/sync/123_back_new&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Metric select</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>281</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="radioLayout">
    <item row="4" column="0">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButton_9">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QRadioButton" name="radioButton_10">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QRadioButton" name="radioButton_11">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QRadioButton" name="radioButton_12">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchronize</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
